docs/user-guide/nextcloud-integration/README.md

```diff
@@ -24,10 +24,10 @@ This integration makes it possible for you to:
 | Topic                                                        | Description                                                  |
 | ------------------------------------------------------------ | :----------------------------------------------------------- |
 | [Connecting your OpenProject and Nextcloud accounts](#connecting-your-openproject-and-nextcloud-accounts) | How to connect your Nextcloud and OpenProject accounts to be able to use this integration |
-| [Linking files and folders to work packages](#linking-files-and-folders-to-work-packages) | How to link/unlink files and folders to work packages and view and download linked files |
-| [Permissions and access control](#permissions-and-access-control) | Who has access to linked files and who doesn't               |
-| [Next steps for the integration](#next-steps-for-the-integration) | What is planned in future releases                           |
-| [Possible errors and troubleshooting](#possible-errors-and-troubleshooting) | Common errors and how to troubleshoot them                   |
+| [Linking files and folders to work packages](#linking-files-and-folders-to-work-packages) | How to link files and folders to work packages and view and download linked files |
+| [Unlinking linked files and folders](#unlinking-files-and-folders) | Unlinking linked files and folders |
```

Comment:
Anchor "#unlinking-files-and-folders" should probably be "#unlinking-linked-files-and-folders" or the corresponding headline be changed to "Unlinking files and folders"

Reply:
Thank you, fixed!